### ISSUE TITLE
How to see Sum-of-Squares decomposotion in the optimization over domain?

### ISSUE THREAD

**Issue by parfenyev** [jump-dev/SumOfSquares.jl#265]
Hello everyone!

I am looking through the example of polynomial optimization over the domain. Just before [the maxdegree keyword argument](https://docs.juliahub.com/SumOfSquares/VZKsW/0.6.3/generated/Polynomial%20Optimization/polynomial_optimization/#The-maxdegree-keyword-argument) subsection, it is explained how the algorithm works. But how can we look at the expressions for polynomials `s0`-`s3` that the algorithm has found? 

Thank you in advance.

**Comment by blegat** [comment]
Yes, you can obtain them with `gram_matrix` and [`lagrangian_multipliers`](https://jump.dev/SumOfSquares.jl/stable/constraints/#SumOfSquares.lagrangian_multipliers)
